libkbfs/tlf_journal.go

```diff
@@ -411,67 +428,52 @@ func (j *tlfJournal) flush(ctx context.Context) (err error) {
 		}
 	}()
 
-	// TODO: Interleave block flushes with their related MD
-	// flushes.
+	// TODO: Avoid starving flushing MD ops if there are many
+	// block ops. See KBFS-1502.
 
-	// Flush the block journal ops in parallel.
 	for {
-		numFlushed, err := j.flushBlockEntries(ctx)
+		blockEnd, mdEnd, err := j.getJournalEnds(ctx)
 		if err != nil {
 			return err
 		}
-		if numFlushed == 0 {
+
+		j.log.CDebugf(ctx, "Flushing up to blockEnd=%d and mdEnd=%d",
```

Comment:
I think this should go below the `break` to avoid weird log messages about flushing to 0.  Maybe have a different log message in the break block saying there is nothing to fluish?